src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/Pivot/BitPivotDemo.razor

```diff
@@ -121,19 +121,22 @@
             <BitPivot LinkFormat="@BitPivotLinkFormat.Tabs"
                       LinkSize="@BitPivotLinkSize.Large"
                       SelectedKey="@OverridePivotSelectedKey"
+                      OnLinkClick="@(item => OverridePivotSelectedKey = item.Key)"
                       SelectedKeyChanged="PivotSelectedKeyChanged">
-                <BitPivotItem HeaderText="Samples">
+                <BitPivotItem Key="1" HeaderText="Samples">
                     Pivot #1
                 </BitPivotItem>
-                <BitPivotItem HeaderText="Files">
+                <BitPivotItem Key="2" HeaderText="Files">
                     Pivot #2
                 </BitPivotItem>
-                <BitPivotItem HeaderText="Recent">
+                <BitPivotItem Key="3" HeaderText="Recent">
                     Pivot #3
                 </BitPivotItem>
             </BitPivot>
-            <BitButton OnClick="(() => OverridePivotSelectedKey = (((int.Parse(OverridePivotSelectedKey) + 3 - 1) % 3)).ToString())">Prev</BitButton>
-            <BitButton OnClick="(() => OverridePivotSelectedKey = (((int.Parse(OverridePivotSelectedKey) + 1 ) % 3)).ToString())">Next</BitButton>
+            <BitButton IsEnabled="@(OverridePivotSelectedKey == "1" ? false : true)"
+                       OnClick="(() => OverridePivotSelectedKey = (((int.Parse(OverridePivotSelectedKey) + 3 - 1) % 3)).ToString())">Prev</BitButton>
+            <BitButton IsEnabled="@(OverridePivotSelectedKey == "3" ? false : true)"
+                       OnClick="(() => OverridePivotSelectedKey = (((int.Parse(OverridePivotSelectedKey) + 1 ) % 4)).ToString())">Next</BitButton>
         </ExamplePreview>
     </ComponentExampleBox>
 
```

Comment:
plz remove all redundant `@` characters, it's only required for the string typed parameters.